setup.py

```diff
@@ -127,7 +189,11 @@ def __init__(
         Extension.__init__(self, name, sources, **kw)
         self.sourcedir = os.path.abspath(cmake_proj_dir)
         self.cmake_flags = cmake_flags or []
-        self.cmake_install_prefix = os.path.abspath("build/cmake_install")
+        self.cmake_install_prefix = (
+            os.path.abspath("build/cmake_install")
+            if not just_extensions
+            else cmake_build_dir
```

Comment:
Wouldn't it be easier to fix `cmake_build_dir` globally to `build/cmake_install` and do away with these inline if statements?